app/services/adapters/structs/income_details.rb

```diff
@@ -33,35 +45,6 @@ def employments
         end
       end
 
-      def client_employment_income
-        income_payments.find do |payment|
-          payment.payment_type == IncomePaymentType::EMPLOYMENT.to_s &&
-            payment.ownership_type == OwnershipType::APPLICANT.to_s
-        end
-      end
-
-      def partner_employment_income
-        income_payments.find do |payment|
-          payment.payment_type == IncomePaymentType::EMPLOYMENT.to_s &&
-            payment.ownership_type == OwnershipType::PARTNER.to_s
-        end
-      end
-
-      def client_work_benefits
-        income_payments.find do |payment|
-          payment.payment_type == IncomePaymentType::WORK_BENEFITS.to_s &&
-            payment.ownership_type == OwnershipType::APPLICANT.to_s
-        end
-      end
-
-      def partner_work_benefits
-        income_payments.find do |payment|
-          payment.payment_type == IncomePaymentType::WORK_BENEFITS.to_s &&
-            payment.ownership_type == OwnershipType::PARTNER.to_s
-        end
-      end
-
-      # TODO: remove businesses exclusion once businesses added
```

Comment:
I don't know what this comment meant, but has it been addressed/no longer needs addressing?